code/modules/admin/holder2.dm

```diff
@@ -61,6 +61,13 @@ var/list/admin_datums = list()
 	if (!admincaster_signature)
 		admincaster_signature = "[current_map.company_name] Officer #[rand(0,9)][rand(0,9)][rand(0,9)]"
 
+/datum/admins/proc/ToggleAOOCDatum()
+	if(toggleantagoocverb == 0)
+		toggleantagoocverb = 1
+		return 0
+	else
+		toggleantagoocverb = 0
+		return 1
```

Comment:
You can just do `return !aooc_toggled` outside of the if instead of two different returns